src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -646,4 +646,10 @@
   <data name="DarkGrayColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
     <value>Dark Gray</value>
   </data>
-</root>
+  <data name="CloseOtherTabsDefaultCommandKey" xml:space="preserve">
+    <value>Close all other tabs</value>
+  </data>
+  <data name="CloseTabsAfterDefaultCommandKey" xml:space="preserve">
+    <value>Close all tabs after</value>

```

Comment:
```suggestion
    <value>Close all tabs after the current tab</value>
```

Maybe? That's maybe a little verbose for the command palette, but the original text seemed a little weird to me

Reply:
If it had to be shorter, I'd say "Close all tabs after current" is fine. But if it fits as is, I think it's fine.

Reply:
![image](https://user-images.githubusercontent.com/11050425/91215571-8d338400-e6c9-11ea-95fb-ece46fe48f15.png)

We're good.